07-regression1.Rmd

```diff
@@ -428,7 +451,7 @@ sacr_summary <- sacr_fit %>%
 sacr_summary
 ```
 Our final model's test error as assessed by RMSPE 
-is `r format(round(sacr_summary %>% filter(.metric == 'rmse') %>% pull(.estimate)), scientific=FALSE)`. 
+is `r format(round(sacr_summary %>% filter(.metric == 'rmse') %>% pull(.estimate)), big.mark=",", nsmall=0, scientific=FALSE)`. 
 But what does this RMSPE score mean? When we calculated test set prediction accuracy in K-NN
```

Comment:
I think we can do better at interpreting here. Our prediction error is ~ 80K on things that range from 70-800K, so can we use that to say something? Also, I wonder if diagnostic plots of the predicted values vs the true values might be useful here? They tell us at least whether we are predicting equally well across the range of the data? They kind of play the same role as a confusion matrix in classification...

Reply:
too nuanced for the book, and we don't cover it in class (no time to)